modules/globebrowsing/src/tileprovider/defaulttileprovider.h

```diff
@@ -57,6 +57,7 @@ class DefaultTileProvider : public TileProvider {
     layers::Group::ID _layerGroupID = layers::Group::ID::Unknown;
     bool _performPreProcessing = false;
     bool _padTiles = true;
+    TileCacheProperties _cacheProperties{ false };
```

Comment:
We prefer to use the `= ` notation since to me this looks more like a function definition.   If the `enabled = false` is set as a default argument in the `struct`, then here we only need `TileCacheProperties _cacheProperties;` anyway